test/src/unit-vfs.cc

```diff
@@ -709,37 +709,42 @@ TEST_CASE(
 
 #ifdef HAVE_AZURE
 TEST_CASE("VFS: Construct Azure Blob Storage endpoint URIs", "[azure][uri]") {
+  // Test the construction of Azure Blob Storage URIs from account name and SAS
+  // token. We are not actually connecting to Azure Blob Storage in this test.
   std::string sas_token, custom_endpoint, expected_endpoint;
   SECTION("No SAS token") {
     sas_token = "";
-    expected_endpoint = "https://devstoreaccount1.blob.core.windows.net";
+    expected_endpoint = "https://exampleaccount.blob.core.windows.net";
```

Comment:
My original thought is to change these endpoints to `http`, which would also disable Entra ID, but we cannot do that, because this test tests the Azure VFS' conversion from account names to endpoints, which always use HTTPS.